### ISSUE TITLE
Thank you for saving Cibyl and Lessphun code on github!

### ISSUE THREAD

**Issue by MadRatSRP** [SimonKagstrom/lessphun#1]

**Comment by SimonKagstrom** [comment]
Well, it's easier than handling the backup yourself isn't it? :-)

If I remember correctly the last development (by me) on cibyl was actually
done on GitHub, but lessphun is just a backup.

**Comment by MadRatSRP** [comment]
I know its been almost 13 years since u're developed for the Mophun, but maybe you can call a few people who can also know something about this platform or you know where can i find source code of mophun games or just a simple example for mophun? We're, the CIS community of those who still are searching information about J2ME and other mobile platforms, want to find more information about Mophun too. Have a nice day :)

**Comment by SimonKagstrom** [comment]
Well, the only Mophun game sources are the ones I wrote myself, which are now in the "old-projects" git repo:  https://github.com/SimonKagstrom/old-projects

As for Mophun itself, I know the main author of it (Anthony Hartley) a bit, and you can probably contact him for more info about it. I have the sources to the GCC port he did, and can mail it to you if you're interested. Not sure how much use it is though, I don't think it's very easy to get it to compile anymore.

**Comment by SimonKagstrom** [comment]
And by the way, the latest Cibyl source code changes are actually here:

https://github.com/sleirsgoevy/cibyl

made by a guy who uses it to port games to PS4 (!). He has fixed at least one Cibyl bug which I wasn't aware of.

**Comment by klaxons1** [comment]
Simon,I wouldn’t refuse the emulator of this platform. I want to play honey cave 2 for a long time:)

**Comment by SimonKagstrom** [comment]
Lessphun is only an API-reimplementation, so it requires the source code.

Mophun uses a GCC backend to produce machine code for their own virtual ISA, so that would have to be emulated in that case :-)

**Comment by klaxons1** [comment]
Too difficult :) Do you know about a mobile platform like BREW?

**Comment by SimonKagstrom** [comment]
No, I only have very slight knowledge about J2ME and Mophun, nothing else.

**Comment by klaxons1** [comment]
There are any tools for decompiling .mpn files ?

**Comment by SimonKagstrom** [comment]
Not that I know of.
